src/cloudflare/internal/images-api.ts

```diff
@@ -40,8 +40,16 @@ class TransformationResultImpl implements ImageTransformationResult {
     return contentType;
   }
 
-  public image(): ReadableStream<Uint8Array> {
-    return this.bindingsResponse.body || new ReadableStream();
+  public image(
+    options?: ImageTransformationOutputOptions
+  ): ReadableStream<Uint8Array> {
+    let stream = this.bindingsResponse.body || new ReadableStream();
```

Comment:
hmm... I know this isn't new but is creating the `new ReadableStream()` like this actually the correct thing to do? This stream looks like it would never be ended/closed

Reply:
This branch should almost never be taken, but replaced it with `new Blob().stream()`